CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 ##
```

Comment:
:fire: ?

Reply:
🤷‍♂️ should I ?

Reply:
It could be problamtic for the changelog parser